examples/terraform/start_localstack.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker run -d -p 4566:4566 -e SERVICES=s3 --name localstack --network="host" localstack/localstack:0.11.5
```

Comment:
We could reuse this file within the Earthfile if we wanted to. Or we could remove it from this example perhaps.